content/fundamentals/unit-testing.md

```diff
@@ -137,6 +137,8 @@ describe('CatsController', () => {
 });
 ```
 
+> info **Hint** `Test.createTestingModule().compile()` does not call life cycle hooks at runtime but they can be called manually for testing purposes
```

Comment:
```suggestion
> info **Hint** `Test.createTestingModule().compile()` does not call lifecycle hooks (i.e., like `onModuleInit` or `onModuleDestroy`) at runtime but they can be called manually for testing purposes.
```

Reply:
Rather than making explicit notes, should we make `lifecycle` link back to the [Lifecycle events](https://docs.nestjs.com/fundamentals/lifecycle-events) page, so that if the names change the documentation doesn't need another update?

Reply:
yeah, that would be a good idea @jmcdo29

Reply:
where it says lifecycle we make a link that leads to the page https://docs.nestjs.com/fundamentals/lifecycle-events ok?

Reply:
That's what I had mentioned @Tony133

Reply:
Do I proceed with the modification?

Reply:
This is my suggestion, and I think what would do well
```suggestion
> info **Hint** `Test.createTestingModule().compile()` does not call [lifecycle hooks](https://docs.nestjs.com/fundamentals/lifecycle-events) at runtime but they can be called manually for testing purposes.
```